keeper/src/main/scala/zio/keeper/membership/package.scala

```diff
@@ -20,6 +20,6 @@ package object membership extends Membership.Service[Membership] {
   override def receive: ZStream[Membership, Error, Message] =
     ZStream.unwrap(ZIO.access[Membership](_.membership.receive))
 
-  override def send(data: Chunk[Byte], receipt: NodeId): ZIO[Membership, Error, Unit] =
-    ZIO.accessM[Membership](_.membership.send(data, receipt))
+  override def send(messageId: String, data: Chunk[Byte], receipt: NodeId): ZIO[Membership, Error, Unit] =
```

Comment:
This is not good idea. this need to be like correlationId in messaging systems. relying on users to make them unique would be silly.

Reply:
I probably misunderstood how it should work. 
I thought that we want to be able to track the same message send by different servers. If every time we send message we will assign it a new correlation_id, we won't be able to track the message across many hops.
But if this is for internal tracking for single send, then ok, I will remove it from the API.

Reply:
we should have something in API like reply to message. then we should keep track of conversationID but we don't have it yet. 

you should look into SWIM and make sure that when we reply for ping and other internal messages we should pass this id along.

Reply:
Ok. I did the changes. Is it ok now? Or should I change the API of all function which send internal message and add this correlactionId there??

Reply:
when you look into method `handleClusterMessages` in `zio.keeper.membership.SWIM` you will see that we are replying for `Ping` and `PingReq` that are part of `SWIM` protocol. Ideally we will use the same `correlationId` for response. Could you add that this?

Reply:
ok